assets/css/print.scss

```diff
@@ -0,0 +1,27 @@
+---
+---
+@charset "utf-8";
+@import "../../do-not-touch/theme-colors/custom.css";
+.bp-section .bp-container .has-side-nav,
+.bp-section .bp-container .has-float-btns,
+.bp-section.bottom-navigation {
+ display: none;
+}
+.bp-section .bp-container .print-content {
+ margin-left: 0px;
+ width: 100%;
+}
+
+@media print {
+  section.bp-section.is-small.bp-section-pagetitle {
+    background: none !important;
+  }
+
```

Comment:
/isomerpages-template/resource_room/resource-room-stuff/opening-speech-by-head-of-civil-service

^the header of this printed page is grey because the "post" layout uses a different header structure. Making an edit and making a PR to you